src/components/Header/Header.tsx

```diff
@@ -1,10 +1,42 @@
-import { Button, Flex } from "@adobe/react-spectrum";
+import { ActionButton, Button, Flex, Item, Menu, MenuTrigger } from "@adobe/react-spectrum";
 import { ToggleButton } from "@adobe/react-spectrum";
+import Home from "@spectrum-icons/workflow/Home";
 import Light from "@spectrum-icons/workflow/Light";
 import Moon from "@spectrum-icons/workflow/Moon";
 import { useAuthenticationContext } from "providers/AuthenticationProvider";
 import { useSignatureContext } from "providers/SignatureProvider";
-import React, { FC } from "react";
+import React, { FC, Key } from "react";
+import { useLocation, useNavigate } from "react-router-dom";
+
+const RouteOptions: FC = () => {
+    // destructuring pathname from useLocation hook
+    const { pathname } = useLocation();
+    const navigate = useNavigate();
+    const selectedPath = pathname.replace("/", "");
+
+    const setSelected = (key: Key) => {
+        navigate(key as string);
+    };
+
+    const capitalizeFirstLetter = (string: string) => {
+        if (string.length === 0) {
+            return <Home />;
+        } else {
+            return string.charAt(0).toUpperCase() + string.slice(1);
+        }
+    };
```

Comment:
method that checks for a string thats not empty. if so returns the home icon if not mutates the first letter to caps and appends to the remaining string with the + string.slice(1);